rct229/data_fns/table_G3_8_fns.py

```diff
@@ -41,7 +41,7 @@
 }
 
 
-def table_G3_8_lpd(building_area_type):
+def table_G3_8_lookup(building_area_type):
     """Returns the lighting power density for a space as
```

Comment:
Lines 53–54 of the documentation needs to be changes to match the new return value.  See the documentation for Table G3.7.